travis/install-sandia-openshmem.sh

```diff
@@ -16,25 +16,24 @@ if [ ! -d "$SHMEM_ROOT" ]; then
     # master
     #git clone --depth 10 https://github.com/regrant/sandia-shmem.git sandia-shmem
     #git clone --depth 10 https://github.com/Sandia-OpenSHMEM/SOS.git sandia-shmem
-    #git clone -b v1.3.0-rc1 --depth 10 https://github.com/Sandia-OpenSHMEM/SOS.git sandia-shmem
+    #git clone -b v1.3.2 --depth 10 https://github.com/Sandia-OpenSHMEM/SOS.git sandia-shmem
     #cd sandia-shmem
     # 1.3 release
-    wget https://github.com/Sandia-OpenSHMEM/SOS/archive/v1.3.0-rc1.tar.gz
-    tar -xzf v1.3.0-rc1.tar.gz
-    cd SOS-1.3.0-rc1
-    # 1.2 release
-    #wget https://github.com/Sandia-OpenSHMEM/SOS/archive/v1.2.0.tar.gz
-    #tar -xzf v1.2.0.tar.gz
-    #cd SOS-1.2.0
+    wget https://github.com/Sandia-OpenSHMEM/SOS/archive/v1.3.2.tar.gz
+    tar -xzf v1.3.2.tar.gz
+    cd SOS-1.3.2
     ./autogen.sh
-    # must build in-place (https://github.com/regrant/sandia-shmem/issues/49)
-    ./configure --with-ofi=$TRAVIS_ROOT/libfabric --with-ofi-libdir=$TRAVIS_ROOT/libfabric/lib \
-                --disable-fortran \
-                --enable-error-checking \
-                --enable-remote-virtual-addressing \
-                --enable-pmi-simple --with-pmi=$TRAVIS_ROOT/hydra \
-                --prefix=$SHMEM_ROOT
-    make && make install
+    mkdir build
+    cd build
+    ../configure --with-libfabric=$TRAVIS_ROOT/libfabric \
+                 --disable-fortran \
+                 --enable-error-checking \
+                 --enable-remote-virtual-addressing \
+                 --enable-pmi-simple --with-pmi=$TRAVIS_ROOT/hydra \
```

Comment:
You should remove the ```--with-pmi=$TRAVIS_ROOT/hydra``` configure argument, as we discussed on the other PR.  It's an error to give both ```--enable-pmi-simple``` and ```--with-pmi=$TRAVIS_ROOT/hydra```.  The ```--with-pmi``` option is for specifying the location of an external PMI client library (e.g. if using SLURM), not the location of the process manager (i.e. Hydra in this case).